helix-view/src/editor.rs

```diff
@@ -1194,14 +1219,14 @@ impl Editor {
         }
     }
 
-    /// Closes language servers with timeout. The default timeout is 500 ms, use
+    /// Closes language servers with timeout. The default timeout is 10000 ms, use
```

Comment:
`10s` seems like a lot and will be noticeable on quit. I think I based 500ms on another implementation

Reply:
The reason I did this is because the Windows CI was failing because it was timing out shutting down.

More broadly speaking, though, 10s is just the upper limit. If your LS closes in < 500ms, then that's all you'll wait. And we should probably try to give the LS a good chance to shut down gracefully, otherwise we risk potentially corrupting the user's workspace state. #3482 is related.

There are so many LSs out there in use on so many different kinds of hardware that it's inevitable that there will be some that almost always take over 500ms to shut down.

Reply:
I find myself waiting for quite some time for it to save, to the point I lost my patience and just `ctrl-z`.

Reply:
@pickfire could you elaborate? This is called when shutting down helix. When are you experiencing it taking a long time? Ctrl-Z would just suspend the process and not help anything.

Reply:
Could you post more details please? Is this happening with this branch and not master? Which LS is this? Do you have any idea why it might be taking 10+ seconds to shut down?